resources/components/ImageCard.vue

```diff
@@ -176,7 +177,20 @@ module.exports = {
 		 * lives in Vuex, so all we need to do is trigger things here.
 		 */
 		onPublish: function () {
-			this.publishTags();
+			var windowManager = new OO.ui.WindowManager();
+
+			// We need to expose confirmTagsDialog on the vm so that we can access it in testing
```

Comment:
Can you add a comment in the code explaining this about more?

Reply:
I have some additional comments in the follow-up PR that enables the "Add custom tag" dialog: https://github.com/egardner/mediawiki-extensions-MachineVision/pull/39/files#diff-492d669d28bf88c47e5aa363e279c44eR186-R234

That PR might be a better place to flesh out the rest of the comments more if needed.